OpenRA.Mods.Common/Effects/FlashTarget.cs

```diff
@@ -38,6 +42,24 @@ public FlashTarget(Actor target, Player asPlayer = null, int count = 2, int inte
 			});
 		}
 
+		public FlashTarget(Actor target, Color color, float alpha, int count, int interval, int delay)
+			: this(target, count, interval, delay)
+		{
+			modifiers = TintModifiers.ReplaceColor;
+			tint = new float3(color.R, color.G, color.B) / 255f;
+			this.alpha = alpha;
+		}
+
+		public FlashTarget(Actor target, float3 tint, int count, int interval, int delay)
+			: this(target, count, interval, delay)
+		{
+			this.tint = tint;
+		}
+
+		// Legacy shim
+		public FlashTarget(Actor target, Player asPlayer = null, int count = 2, int interval = 2, int delay = 0)
+			: this(target, asPlayer?.Color ?? Color.White, 0.5f, count, interval, delay) { }
```

Comment:
`Legacy shim` sounds like this is something we want to get rid off?

Reply:
Yes in the sense that this old api isn't great. No in the sense that I didn't want to rewrite all the other callers as part of this PR.

Updated with a middle approach that should hopefully be fine.